6-Typescript/server/tests/it/get-all-todos.test.ts

```diff
@@ -15,7 +15,7 @@ describe("GET all todos /api/todos/ ", () => {
         expect(response.data).toEqual([]);
     });
 
-    it("given todos in db should return array of todos and status 200", async () => {
+    it("should return array of todos and status 200, given todos in db", async () => {
         const todos: ITodo[] = times(3, () => createTodo());
         await Promise.all(
```

Comment:
this can be move to a driver (I don't be aware of the fact that you use MongoDB)
and - your `Promise.all` is a part of the given.

so you can do something like 
`driver.given.todos(todos)`
when in your driver - you'll do the `await Promise.all...`

Reply:
regarding the `setUserId` - what is special in `todos[0].userId`? (probably nothing but the fact the it exists in one of the todos)

Reply:
ok - now I get it - your test doesn't check `should return array of todos and status 200, given todos in db`

it actually checks that given many todos (in your case - 3) - a user gets the todos that are relevant only for him (`todos[0]` for `todos[0].userId`)
please fix it

Reply:
regarding your first comment - i wrote it like that because this is what we had in our TDD katas with the backend team.
Including user the name of mongoDB. This is how it looked like there:
"
    const users = times(5, aUser);

    await Promise.all(users.map((user) => drivers.mysql.insertUser(user)));

    const response = await drivers.app.listUsers();
"

Regarding your next comments - ill change the "it" that it will be more accurate about the specific user.